### PR TITLE
Feature/110

### LINKED ISSUE #110: [FEATURE] - Create Student
### Reference Issues
#108 

### Summary
- Create an endpoint for registering as a student user.
- Create appropriate DTO and validation for the user.

### Definition of Done
- When the above is implemented and tested.

### PR DESCRIPTION
**Pull request by KristianS93**
## Description

Update a registration endpoint, seed an instructor account. 

### Resolved Issue
Fixes #110

## Changes

Changes to register endpoint and seeding of instructor account. 
This depends on the subtask #108 and #123 have to be merged before. 

## Checklist

- [X] New and old tests passed
- [X] Documentation is updated

### CODE REVIEW

File: API/Properties/launchSettings.json [hunk 1]
```diff
@@ -18,7 +18,11 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development",
         "JWT_KEY": "8a6e4bde-3a4b-4691-9223-1662e4d3b4c4",
-        "MOZART_HASKELL": "localhost:8080"
+        "MOZART_HASKELL": "localhost:8080",
+        "SEED_ADMIN": "true",
+        "ADMIN_MAIL": "admin@p7.dk",
```
Comment (andreaswurtz) [inline API/Properties/launchSettings.json:23]: this will save many bruno clicks

File: Core/Shared/Roles.cs
Comment (andreaswurtz) [inline Core/Shared/Roles.cs:1]: RolesConvert best class

File: Infrastructure/Authentication/UserService.cs
Comment (andreaswurtz) [inline Infrastructure/Authentication/UserService.cs:1]: How to make instructor now?
image

**Review by andreaswurtz — APPROVED** [review]
🏳️‍🌈

File: Infrastructure/Authentication/UserService.cs
Comment (KristianS93) [reply Infrastructure/Authentication/UserService.cs:1]: you CANT

Comment (andreaswurtz) [reply Infrastructure/Authentication/UserService.cs:1]: rip instructor

**Pull request merged**